### ISSUE TITLE
Set support

### ISSUE THREAD

**Issue by Alexsey** [shouldjs/should.js#162]
Hi

For now this code
```js
require('should')

const s = new Set([1, 2, 3])
s.should.containEql(1)
```
would throw an error
```
Error: There is no type adaptor `forEach` for number
```

While this
```js
require('should')

const a = [1, 2, 3]
a.should.containEql(1)
```
would work as expected

Probably `Map` support should come in a bundle with `Set` support although `Set` support would also be really good to have

BTW in `chai` it works ;)

**Comment by btd** [comment]
Fixed in 13.2.0

**Comment by Alexsey** [comment]
Cool! Thanks!
